CMakeLists.txt

```diff
@@ -584,24 +584,48 @@ set(OPENSIM_DEPENDENCIES_DIR
     "Directory containing installed binaries of OpenSim dependencies. Set this
      only if you used the Superbuild procedure to install dependencies. ")
 
-set(WITH_BTK
-    OFF
-    CACHE
-    BOOL
-    "Compile OpenSim with BTK ? BTK provides C3D reading.")
-
-# If compiling with BTK, find and use it.
-if(WITH_BTK)
-    find_package(BTK 
+set(C3D_PARSER None CACHE STRING
```

Comment:
@pariterre can you change this to something like `OPENSIM_C3D_PARSER` to be consistent with some other CMake options/settings?

Reply:
I plan to make a PR to do this renaming.